modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/pages/apps/edit/EditApp.es.js

```diff
@@ -111,113 +102,23 @@ export default ({
-												'create-one-or-more-forms-to-display-the-data-held-in-your-data-object'
-											),
-											Liferay.Language.get(
-												'there-are-no-form-views-yet'
-											)
-										)}
-										endpoint={`/o/data-engine/v2.0/data-definitions/${dataDefinitionId}/data-layouts`}
-										itemType="DATA_LAYOUT"
-										title={
-											<h2>
-												{Liferay.Language.get(
-													'select-a-form-view'
-												)}
-											</h2>
-										}
-									/>
-								)}
-
-								{currentStep == 1 && (
-									<EditAppBody
-										defaultLanguageId={defaultLanguageId}
-										emptyState={getEmptyState(
-											Liferay.Language.get(
-												'create-one-or-more-tables-to-display-the-data-held-in-your-data-object'
-											),
-											Liferay.Language.get(
-												'there-are-no-table-views-yet'
-											)
-										)}
-										endpoint={`/o/data-engine/v2.0/data-definitions/${dataDefinitionId}/data-list-views`}
-										itemType="DATA_LIST_VIEW"
-										title={
-											<h2>
-												{Liferay.Language.get(
-													'select-a-table-view'
-												)}
-											</h2>
-										}
-									/>
-								)}
-
-								{currentStep == 2 && (
-									<EditAppBody
-										defaultLanguageId={defaultLanguageId}
-										emptyState={{
-											search: {
-												className:
-													'taglib-search-state',
-												title: Liferay.Language.get(
-													'no-results-were-found'
-												),
-											},
-										}}
-										endpoint={`/o/headless-admin-workflow/v1.0/workflow-definitions?active=true&page=-1&pageSize=-1`}
-										itemType="WORKFLOW_PROCESS"
-										title={
-											<>
-												<h2>
-													{Liferay.Language.get(
-														'connect-a-workflow'
-													)}
-													<span className="text-secondary">
-														{` (${Liferay.Language.get(
-															'optional'
-														)})`}
-													</span>
-												</h2>
-
-												<span className="text-secondary">
-													{Liferay.Language.get(
-														'enable-app-submissions-to-flow-through-a-workflow-process'
-													)}
-												</span>
-											</>
-										}
-									/>
-								)}
-
-								{currentStep == 3 && <DeployApp />}
-							</div>
-
-							<h4 className="card-divider"></h4>
-
-							<EditAppFooter
+							<EditAppBody
 								currentStep={currentStep}
+								dataDefinitionId={dataDefinitionId}
 								defaultLanguageId={defaultLanguageId}
-								editingLanguageId={editingLanguageId}
-								onCurrentStepChange={(step) => {
-									setCurrentStep(step);
-								}}
 							/>
 						</div>
+
+						<h4 className="card-divider"></h4>
+
+						<EditAppFooter
+							currentStep={currentStep}
+							defaultLanguageId={defaultLanguageId}
+							editingLanguageId={editingLanguageId}
+							onCurrentStepChange={(step) => {
+								setCurrentStep(step);
+							}}
```

Comment:
Hey @gabrielpramos, what do you think about just call the setCurrentStep instead of creating a new anonymous function?

Something like: `onCurrentStepChange={setCurrentStep}`

Reply:
sure, this will be sent in a future [pr](https://github.com/gabrielpramos/liferay-portal/pull/136)